openapi/paths/payment-methods.yaml

```
@@ -1,170 +1,43 @@
-                      type: boolean
-                      example: true
-                    country_preferred:
-                      description: The merchant's preferred country for using this payment method. 
-                      example:
-                        - NL
-                      items:
-                        type: string
-                      type: array
-                    direct_available:
-                      description: |
-                        Whether the payment method supports `type`: `direct` orders.
-                      type: boolean
-                    direct_preferred:
-                      description: |
-                        Whether the merchant prefers `type`:`direct` orders for this payment method.
-                      type: boolean
-                    extras:
-                      description: Additional information about issuers for the payment method.
-                      type: object
-                      example:
-                        issuers:
-                          - bic: ABNANL2A
-                            code: '0031'
-                            name: ABN Amro Bank
-                    icon_urls:
-                      description: URLs to versions of the payment method logo.
-                      properties:
-                        large:
-                          description: URL to a large-sized version of the payment method logo.
-                          type: string
-                          example: >-
-                            https://media.multisafepay.com/img/methods/3x/visa.png
-                        medium:
-                          description: URL to a medium-sized version of the payment method logo.
-                          type: string
-                          example: >-
-                            https://media.multisafepay.com/img/methods/2x/visa.png
-                        vector:
-                          description: URL to a vector version of the payment method logo.
-                          type: string
-                          example: >-
-                            https://media.multisafepay.com/img/methods/svg/visa.svg
-                      type: object
-                    id:
-                      description: The identifier of the payment method.
-                      type: string
-                      example: VISA
-                    name:
-                      description: The name of the payment method.
-                      type: string
-                      example: Visa
-                    required_fields:
-                      example: []
-                      items:
-                        type: string
-                      type: array
-                  type: object
-                type: array
               success:
-                type: boolean
-            type: object
-      description: ''
+                $ref: '../components/schemas/Success.yaml'
+              data:
```

```suggestion
              data:
                 description: Array of payment methods
                 type: array
                 items: 
                   $ref: '../components/schemas/PaymentMethod.yaml'
```
An array of payment methods is returned